applications/ColossalQA/data/tests/test.md

```diff
@@ -0,0 +1,78 @@
+# README Format File for Testing
+![Alt text](./examples/diagram.png?raw=true "Fig.1. design of the document retrieval conversation system")
```

Comment:
Link here seems wrong

Reply:
this is only a markdown format file for testing the data loader module as indicated in line 1